cpuid_x86.c

```diff
@@ -1955,6 +1955,19 @@ int get_coretype(void){
 	    return CORE_NEHALEM;
         }
         break;
+      case 10:
+        switch (model) {
+    case 6:
+      // Comet Lake U
+            if(support_avx())
+  #ifndef NO_AVX2
+              return CORE_HASWELL;
```

Comment:
I think it needs to return CPUTYPE_HASWELL in other place.

Reply:
Yep, it was my oversight I didn't realize there are 2 places to update. Added in: https://github.com/xianyi/OpenBLAS/pull/2683/commits/2f9c10810c932fc015cb4e5078cab7117bc120b6